### PR TITLE
feat: add min max size for random slice map And allowing one element for Oneof

### PR DESCRIPTION
**Pull request by autumn31**
wish to add a feature that can set minimum random size for slice and map

### CODE REVIEW

**Comment by bxcodec** [comment]
Hi @autumn31  can you help to check the test, it's failing on the CI

Sorry for the late reply, been busy lately. 🙇🏻

**Comment by autumn31** [comment]
Test fixed. Also I updated the mr title, since I added another feature that `Oneof should accept "one element" cases`.
Wonder how you think about this change?

File: faker.go [hunk 1]
```diff
@@ -1087,6 +1090,26 @@ func extractNumberFromText(text string) (int, error) {
...
+func fetchOneOfArgsFromTag(tag string) ([]string, error) {
+	items := strings.Split(tag, colon)
+	argsList := items[1:]
+	if len(argsList) != 1 {
+		return nil, fmt.Errorf(ErrUnsupportedTagArguments)
+	}
+	if strings.Contains(argsList[0], ",,") {
+		return nil, fmt.Errorf(ErrDuplicateSeparator)
+	}
+	fmt.Printf("argsList(%v)\n", argsList[0])
```
Comment (bxcodec) [inline faker.go:1102]: can remove this?

**Review by bxcodec — COMMENTED** [review]
Hey @autumn31 

So sorry for the late reply. 
Minor comment, otherwise LGTM

File: faker.go (hunk 1 above)
Comment (autumn31) [reply faker.go]: Sure, this line was removed in the latest commit.

**Review by bxcodec — APPROVED** [review]
LGTM

**Comment by bxcodec** [comment]
Hey @autumn31 , 

Thanks for the PR, I'll release it soon. In the mean time, you can try to test on your project using the master branch for the source

**Pull request merged**